core/repository/manager/src/test/java/org/eclipse/rdf4j/repository/manager/RepositoryManagerTest.java

```diff
@@ -36,7 +36,7 @@ public class RepositoryManagerTest {
 
 	protected RepositoryManager subject;
 
-	@Before
+	@BeforeEach
```

Comment:
Should this test class be redeclared as abstract and this setUp method deleted? I think so, but there might be a reason for having it like this? A bit confusing now with methods that are semi-overidden....